variables.tf

```diff
@@ -39,6 +39,12 @@ variable "frontend_subnet_id" {
   default     = ""
 }
 
+variable "lb_enable_floating_ip" {
```

Comment:
Since all `enable_xxx` arguments would be renamed to `xxx_enabled` in AzureRM provider 4.0, could we rename this variable to `lb_floating_ip_enabled`?